src/ssl.c

```diff
@@ -12451,6 +12487,7 @@ int wolfSSL_EVP_MD_type(const WOLFSSL_EVP_MD *md)
             return NULL;
     #endif
 
+        XMEMSET(hmac, 0, sizeof(Hmac));
```

Comment:
Yes, the memset here is required because of new items in the structure. I've added an wc_HmacInit function already, but wasn't calling it here. This has been fixed. I also added a memset to wc_HmacInit.